src/promise-katas-2.js

```diff
@@ -25,17 +25,40 @@ fetch("joke", "question").then(<insert your callback function>)
 // 1 Create a function that uses the fetch function to make a request to the "food" URL and returns
 // the data - expected return value "Cheese" of type String
 
-const food = () => {};
+const food = async () => {
+  const response = await fetch('food')
+  console.log(response)
```

Comment:
You should remove console.logs from the file if they are not part of the API and were put there for debugging purposes